code/game/machinery/Sleeper.dm

```diff
@@ -360,9 +361,9 @@
 
 /obj/machinery/sleeper/blob_act()
 	if(prob(75))
-		for(var/atom/movable/A as mob|obj in src)
-			A.forceMove(src.loc)
-			A.blob_act()
+		var/atom/movable/A = src.loc

```

Comment:
This is similar to the other line note and looking again, should target the occupant as well.